README.md

```diff
@@ -26,33 +26,126 @@ Analysis” locally.
 
 ## Quickstart
 
-We recommend using [Poetry](https://python-poetry.org/) for installing the tool
-and working in a virtual environment. Once you have Poetry set up:
+There are 2 main ways to use the brokenspoke-analyzer:
+
+- All in Docker
+- Native Python with the database running in a Docker container
+
+The different methods are being described in the sections bellow, with their
+advantages and inconvenients.
+
+For more details about the different ways to run an analysis and how to adjust
+the options, please refer to the full documentation.
+
+### All in Docker
+
+The benefit of running eveything using the provided Docker images, is that there
+is no need to install any of the required dependencies, except Docker itself of
+course. This guaranties that the user will have the right versions of the
+multiple tools that are combined to run an analysis. This is the simplest way,
+and the recommended way for people who just want to run the analyzer.
+
+Start the database from Docker Compose, in the background:
 
 ```bash
-git clone git@github.com:PeopleForBikes/brokenspoke-analyzer.git
-cd brokenspoke-analyzer
-poetry install
+docker compose up -d
+```
+
+Export the database URL:
+
+```bash
+export DATABASE_URL=postgresql://postgres:postgres@localhost:5432/postgres
+```
+
+Run the analysis:
+
+```bash
+docker run \
+  --rm \
+  --network brokenspoke-analyzer_default \
+  -e DATABASE_URL \
+  ghcr.io/peopleforbikes/brokenspoke-analyzer:2.4.0 \
```

Comment:
Just wanted to make sure, we have version 2.4.0 listed because we will be making a release that includes the Docker fix (#729) and a few other ones since 2.3.0, right?

Reply:
Correct. This documentation will be online at the same time as the 2.4.0 release. 👍